openmoc/subdivider.py

```diff
@@ -0,0 +1,124 @@
+# Subdividing Lattice
+#
+# Class that performs spatial discretization on a mesh
```

Comment:
other Python files don't have a header.
If we're going to have one, let's have a clean nice one like for .cpp files